src/datacontroller.js

```diff
@@ -204,4 +212,31 @@ export default class DataController {
 	 * Removes all event listeners set by the DataController.
 	 */
 	destroy() {}
+
+	/**
+	 * See {@link engine.model.composer.insertContent}.
+	 *
+	 * @fires engine.DataController#insertContent
+	 * @param {engine.model.Batch} batch Batch to which deltas will be added.
+	 * @param {engine.model.Selection} selection Selection into which the content should be inserted.
+	 * @param {engine.view.DocumentFragment} content The content to insert.
+	 */
+	insertContent( batch, selection, content ) {
```

Comment:
To make it similar to other methods here we could rename it to simple `insert`. It will create then a nice API: `editor.data.insert()`. Also, the order of parameters should be same as other methods: what, where, other. Note that `content` is the only parameter which for sure will not be optional. So: `insert( content, selection, batch )`

Moreover, we may make the API more consistent: all `parse`, `set` and `insert` could get data as string or view and call `data = this.processor.toView( data );` if data is a string. 

Also, we have a different policy when it comes to the `batch` in `set` and `insert`. I think both could take `batch` as an optional parameter and create a batch if it is not passed.

It would be a nice and useful API, not only for clipboard: `editor.data.insert( '<p>foo</p>', editor.document.selection )`.